CONTRIBUTE.md

```diff
@@ -0,0 +1,239 @@
+# Contributing to Umple
+
+Please take a moment to review this document in order to make the contribution
+process easy and effective for everyone involved!
+
+## Using the issue tracker
+
+Use the issues tracker for:
+
+* [bug reports](#bug-reports)
+* [submitting pull requests](#pull-requests)
+
+Please **do not** use the issue tracker for personal support requests nor feature requests. Support requests should be sent to:
```

Comment:
Line 13 should not say 'feature requests'. Indeed we want the issue tracker to be used for feature requests.